<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edit</class>
 <widget class="QWidget" name="Edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>588</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>588</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Добавление/удаление видов</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 179, 164);</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>140</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>Закончить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Введите данные о виде, который хотите добавить</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>199</width>
     <height>198</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Класс</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Отряд</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Семейство</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Вид</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>85</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineClass">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(230, 230, 230);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineOtrad">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(230, 230, 230);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineSemestvo">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(230, 230, 230);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineVid">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(230, 230, 230);</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>80</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>81</y>
     <width>210</width>
     <height>72</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Выберите вид, который хотите удалить</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(230, 230, 230);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(230, 230, 230);</string>
        </property>
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
